dev/TreeView/TreeViewListAutomationPeer.cpp

```diff
@@ -44,4 +45,26 @@ winrt::IInspectable TreeViewListAutomationPeer::GetPatternCore(winrt::PatternInt
 winrt::AutomationControlType TreeViewListAutomationPeer::GetAutomationControlTypeCore()
 {
     return winrt::AutomationControlType::Tree;
-}
\ No newline at end of file
+}
+
+// ISelectionProvider
+bool TreeViewListAutomationPeer::CanSelectMultiple()
+{
+    return IsMultiselect()? true: __super::CanSelectMultiple();
```

Comment:
Why call the base if we aren't multiselect? Why not just say "no"?

Reply:
for single select, we use the base listview's selection logic.

---
In reply to: [249943672](https://github.com/Microsoft/microsoft-ui-xaml/pull/210#discussion_r249943672) [](ancestors = 249943672)